tailscale/resource_device_authorization.go

```diff
@@ -33,7 +36,7 @@ func resourceDeviceAuthorization() *schema.Resource {
 
 func resourceDeviceAuthorizationRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	client := m.(*tailscale.Client)
-	deviceID := d.Get("device_id").(string)
+	deviceID := d.Id()
```

Comment:
This is required for import to function as the `device_id` is not set when we are importing, although they are equivalent values.